app/config/ErrorHandler.scala

```diff
@@ -36,7 +36,7 @@ class ErrorHandler @Inject()(val messagesApi: MessagesApi, errorTemplate: error)
   override def resolveError(request: RequestHeader, exception: Throwable): Result = exception match {
     case ReturnToStartException =>
       logger.warn(s"User Answers was in an invalid state, returning them to the Start Page from [${request.uri}]")
-      Results.Redirect(controllers.routes.ChoiceController.displayPage())
+      Results.Redirect(controllers.ileQuery.routes.IleQueryController.submitQueryForm())
```

Comment:
This is incorrect. AFAIR user should be redirected to `displayQueryForm` method in the same controller.

Reply:
Return to start must redirect to the original choice page.